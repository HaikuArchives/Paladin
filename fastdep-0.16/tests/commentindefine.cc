#ifndef symbol		/* some comment */
#include "a.h"
#endif
