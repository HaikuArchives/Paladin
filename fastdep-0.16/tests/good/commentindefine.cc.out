commentindefine.o: commentindefine.cc \
	a.h \
	c.h \
	d.h
